test/harness/network.go

```diff
@@ -6,11 +6,11 @@ import (
 	"github.com/orbs-network/orbs-network-go/bootstrap"
 	"github.com/orbs-network/orbs-network-go/config"
 	"github.com/orbs-network/orbs-network-go/instrumentation"
+	stateStorageAdapter "github.com/orbs-network/orbs-network-go/services/statestorage/adapter"
```

Comment:
I think I've talked to Ron about this. 
It's bad practice to include the production state storage adapter in the test harness. It would be better to duplicate the adapter file. @electricmonk Thoughts?

Reply:
not yet. It's better not to do that at this moment, or even move this code to be only production code.

Reply:
I'm not following, @electricmonk  do you mean the way it is implemented now is preferable or that we should move this adapter to the test harness?

Reply:
I think that currently there's only one persistence adapter and it is a production one. Don't move to test harness. If/when we end up creating a persistent implementation, this one will have to move to test harness and a contract test will be required to assert that both implementations conform to the same contract.